src/patternfly/components/BackgroundImage/styles.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../patternfly-utilities";
+
+.pf-c-background-image {
+  &::before {
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: -1;
+    width: 100vw;
+    height: 100vh;
+    content: "";
+    background: url("#{$pf-global--image-path}/pfbg_576.jpg") no-repeat;
```

Comment:
how about separating this out to `background-image: url("#{$pf-global--image-path}/pfbg_576.jpg")` and `backround-repeat: no-repeat;`

Then just updating `background-image: url(...)` in the `@media` queries below, and removing `background-attachment: fixed;` from the media queries since it's defined in the base css.

Reply:
as far as I know for background properties you need to state them for every MQ for some reason. I could be wrong.